packages/website/private/contents/post-006.mdx

```diff
@@ -0,0 +1,60 @@
+---
+slug: type-safe-routing
+title: Type-safe routing
+description: Enjoy type safety for routing and navigation APIs in Waku projects.
+author: sophia
+date: 2024/12/10
+---
+
+Continuing our holiday retrospective of Waku’s recent features, today we’re excited to highlight type-safe routing. This addition brings compile-time safety to your routing and navigation logic while staying true to Waku’s minimalist principles.
+
+## Type-safe routing
+
+### Configuration
+
+While support for loose typing comes out of the box, you can opt into further type inference in TypeScript projects by adding `as const` assertions to the return of `getConfig` for each of your project’s routes.
+
+```tsx
+// ./src/pages/valid/[slug].tsx
+
+export default async function SomePage() { ... }
+
+export const getConfig = async () => {
+  const staticPaths = await getStaticPaths();
+
+  return {
+    render: 'static',
+    staticPaths,
+  } as const; // NEW!
+};
```

Comment:
```suggestion
export const getConfig = async () => {
  return {
    render: 'static',
    staticPaths: ['foo', 'bar'],
  } as const; // NEW!
};
```
could we do the example like this? It's less "real world", but since we don't fill `getStaticPaths` here, it will show a more type-safe usage as the example.